app/views/foreman_bootdisk/host.erb

```diff
@@ -31,7 +30,9 @@ echo Configuring net${idx} for static IP address <%= interface.ip %>
 ifopen net${idx}
 set netX/ip <%= interface.ip %>
 set netX/netmask <%= interface.subnet.mask %>
+<% if interface.subnet.gateway %>
```

Comment:
`.present?` to also catch an empty value?

Reply:
Done!